integration-tests/models/test_flash_qwen2_vl_video.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import json
+import requests
+
+
+@pytest.fixture(scope="module")
+def qwen2_vl_handle(launcher):
+    with launcher(
+        "Qwen/Qwen2-VL-7B-Instruct",
+        max_input_length=10_000,
+        max_batch_prefill_tokens=10_000,
+        max_total_tokens=10_001,
+        cuda_graphs=[0],
```

Comment:
Remove all of these. We shouldn't need them.
We NEED to make sure users don't have to set anything.

Reply:
agreed removed in latest commit